packages/common/src/components/CondaPkgList.tsx

```diff
@@ -143,6 +147,23 @@ export class CondaPkgList extends React.Component<IPkgListProps> {
       return <span>{rowData.name}</span>;
     };
 
+    const versionRender = ({ rowData }: ICellRender): JSX.Element => (
+      <a
+        className={
+          rowData.updatable
+            ? classes(Style.Updatable, Style.Link)
+            : Style.Link
+        }
+        href="#"
+        onClick={(evt): void => {
+          this.props.onPkgGraph(rowData);
+        }}
+        rel="noopener noreferrer"
```

Comment:
```suggestion
        rel="noopener noreferrer"
        title="Show dependency graph"
```

It would be nice to add a title to inform the user of the action resulting of the click.